platform/android/app/src/main/kotlin/coop/polypoly/polypod/MainActivity.kt

```diff
@@ -34,13 +34,11 @@ class MainActivity : AppCompatActivity(), LifecycleEventObserver {
             logger.info("Core is bootstrapped!")
         } catch (ex: Exception) {
             logger.info(ex.message)
-            (ex as? CoreFailure)?.also {
-                // Ignore CoreAlreadyBootstrapped error, as it is not breaking.
-                if (it.code == CoreExceptionCode.CoreAlreadyBootstrapped) {
-                    return
-                }
-            }
-            throw ex
+            // Ignore CoreAlreadyBootstrapped error, as it is not breaking.
+            if ((ex as? CoreFailure)?.code
+                != CoreExceptionCode.CoreAlreadyBootstrapped
+            )
```

Comment:
Pretty odd formatting, but I arrived at this through trial and error - every other way I found that makes ktlint happy looks uglier, so...